durable-task-step/src/main/resources/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStep/help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Runs a Bourne shell script on a Unix node using <code>nohup</code>. Multiple lines are accepted.
```

Comment:
What you *should* talk about is the fact that you may use

```perl
#!/usr/bin/perl
```

syntax to select an alternate interpreter, and also that you may want to run

```bash
set +e
```

and/or

```bash
set +x
```

to disable the flags which are set by default.